app/graphql/mutations/payment_providers/adyen.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mutations
+  module PaymentProviders
+    class Adyen < BaseMutation
```

Comment:
Is it possible to add a spec file for this class?

Something in `spec/graphql/mutations/payment_providers/ayden_spec.rb`